android/storage.md

```diff
@@ -108,51 +110,73 @@ The TransferUtility checks the size of the file being uploaded and automatically
+import com.amazonaws.mobile.client.UserStateDetails;
 import com.amazonaws.mobileconnectors.s3.transferutility.TransferListener;
+import com.amazonaws.mobileconnectors.s3.transferutility.TransferObserver;
+import com.amazonaws.mobileconnectors.s3.transferutility.TransferService;
+import com.amazonaws.mobileconnectors.s3.transferutility.TransferState;
+import com.amazonaws.mobileconnectors.s3.transferutility.TransferUtility;
 import com.amazonaws.services.s3.AmazonS3Client;
 
+import java.io.BufferedWriter;
 import java.io.File;
+import java.io.FileWriter;
 
-public class YourActivity extends Activity {
-
-    private static final String TAG = YourActivity.class.getSimpleName();
+public class MainActivity extends Activity {
+    private static final String TAG = MainActivity.class.getSimpleName();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        getApplicationContext().startService(new Intent(getApplicationContext(), TransferService.class));
+
         // Initialize the AWSMobileClient if not initialized
         AWSMobileClient.getInstance().initialize(getApplicationContext(), new Callback<UserStateDetails>() {
             @Override
             public void onResult(UserStateDetails userStateDetails) {
                 Log.i(TAG, "AWSMobileClient initialized. User State is " + userStateDetails.getUserState());
+                uploadWithTransferUtility();
             }
 
             @Override
             public void onError(Exception e) {
                 Log.e(TAG, "Initialization error.", e);
             }
         });
-        uploadWithTransferUtility();
+
     }
 
     public void uploadWithTransferUtility() {
 
         TransferUtility transferUtility =
-            TransferUtility.builder()
-                .context(getApplicationContext())
-                .awsConfiguration(AWSMobileClient.getInstance().getConfiguration())
-                .s3Client(new AmazonS3Client(AWSMobileClient.getInstance()))
-                .build();
+                TransferUtility.builder()
+                        .context(getApplicationContext())
+                        .awsConfiguration(AWSMobileClient.getInstance().getConfiguration())
+                        .s3Client(new AmazonS3Client(AWSMobileClient.getInstance()))
+                        .build();
+
+        File file = new File(getApplicationContext().getFilesDir(), "sample.txt");
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter(file));
+            writer.append("Howdy World!");
+            writer.close();
+        }
+        catch(Exception e) {
+            Log.e(TAG, e.getMessage());
+        }
 
         TransferObserver uploadObserver =
-            transferUtility.upload(
-                "public/s3Key.txt",
-                new File("/path/to/file/localFile.txt"));
+                transferUtility.upload(
```

Comment:
please confirm that there are only spaces, no tabs, in the file

Reply:
Confirmed they are spaces not tabs